src/site/content/en/blog/yahoo-japan-identity-case-study/index.md

```diff
@@ -0,0 +1,426 @@
+---
+layout: post
+title: Yahoo! JAPAN's password-free authentication reduced inquiries by 25%, sped up sign-in time by 2.6x
+subhead: >
+    Learn about how Yahoo! Japan built a passwordless identity system.
+description: |
+    Yahoo! JAPAN achieved a passwordless identity system. Learn their approach and the consequences of the effort.
+authors:
+  - yuyaito
+  - agektmr
+  - alexandrawhite
+date: 2022-05-10
+hero: image/VbsHyyQopiec0718rMq2kTE1hke2/tqK2Tb4rUrxheagyYbDj.png
+alt:
```

Comment:
this is why presubmit is failing.

Reply:
Oh sure, fun! Waiting for my update to push.

Reply:
The preview may still be failing -- any insights as to why?